CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set(JSON_GIT_TAG          v3.7.3)
 set(XEUS_GIT_TAG          0.24.1)
 set(PYBIND11_GIT_TAG      v2.3)
 set(PYBIND11_JSON_GIT_TAG 0.2.6)
-set(XEUS_PYTHON_GIT_TAG   0.8.3)
```

Comment:
I think it makes sense to update xeus to 0.24.2 as well.

Reply:
Good catch :+1: